src/components/organisms/UiDatepicker/_internal/UiDatepickerTabItem.vue

```diff
@@ -22,6 +22,7 @@ export type DatepickerTabItemAttrsProps = DefineAttrsProps<null, ButtonAttrsProp
 
   @include mixins.override-logical(button, $element, padding, var(--space-8));
   @include mixins.override-logical(button, $element, border-width, 0);
+  @include mixins.override-logical(button, $element, border-radius, var(--space-8));
```

Comment:
```suggestion
  @include mixins.override-logical(button, $element, border-radius, var(--border-radius-form));
```
We could consider using token instead of raw value here.

Reply:
Good idea. Implemented it.